src/Tribe/Tickets_View.php

```diff
@@ -979,6 +979,12 @@ public function get_tickets_block( $post ) {
 		/** @var \Tribe__Tickets__Editor__Blocks__Tickets $blocks_tickets */
 		$blocks_tickets = tribe( 'tickets.editor.blocks.tickets' );
 
+		// load assets if classic editor is active
+		$option = get_option( 'classic-editor-replace' );
```

Comment:
- I am not sure if this check is required.
- We could always load the assets here as there are other plugins to disable gutenberg besides the classic editor. 
- Not sure if there is a scenario where we load assets twice and it creates a performance issue.

Reply:
It sounds like we should always load the assets (once) here. Doesn't the assets loader only ever load it once?

Reply:
I believe so. I updated the PR to always load them. Ready for another review.